config/schemas/config.yaml

```diff
@@ -1198,107 +1198,277 @@ properties:
+        type: string
+        enum:
+          - azure
+          - gcs
+          - s3
+          - none
+      azure:
         additionalProperties: false
+        title: Azure Backend Config
+        description: Only supports Azure Public Cloud.
+        type: object
         properties:
-          accessKey:
-            type: string
-          forcePathStyle:
-            description: Generally false when using AWS and Exoscale and true for other providers.
-            type: boolean
-          region:
-            type: string
-          regionEndpoint:
-            description: S3 endpoint for non-AWS implementation of S3. Make sure to prepend the protocol (i.e., https:// or http://) with the URL, e.g., https://s3.sto1.safedc.net
-            type: string
-          secretKey:
+          resourceGroup:
+            description: Resource group of the storage account.
             type: string
-        type: object
-      sync:
+      s3:
         additionalProperties: false
+        title: S3 Backend Config
+        description: Supports both AWS and non-AWS implementations.
+        type: object
         properties:
-          buckets:
-            type: array
-          defaultSchedule:
-            default: 0 5 * * *
+          region:
+            description: S3 region.
             type: string
-          destinationType:
-            default: s3
+          regionEndpoint:
+            description: |-
+              Endpoint to reach the S3 service, mainly applicable for non-AWS implementations.
+
+              Make sure to prepend the protocol (e.g. `https://`).
             type: string
-          enabled:
-            default: false
-            type: boolean
-          encrypt:
-            additionalProperties: false
-            properties:
-              directoryNames:
-                default: false
-                type: boolean
-              enabled:
-                default: false
-                type: boolean
-              fileNames:
-                default: false
-                type: boolean
-            type: object
-          resources:
-            $ref: '#/$defs/kubernetesResourceRequirements'
-          s3:
-            $ref: '#/properties/objectStorage/properties/s3'
-          syncDefaultBuckets:
-            default: false
+            format: uri
+          forcePathStyle:
+            description: |-
+              Force the use of path style access instead of virtual host style access.
+
+              Generally `false` when using AWS, Exoscale, and UpCloud and `true` for other providers.
             type: boolean
-        type: object
-      type:
-        enum:
-          - s3
-          - gcs
-          - none
-        type: string
       swift:
+        additionalProperties: false
+        title: Swift Backend Config
+        description: |-
+          > [!note]
+          > Supported as an option only for Harbor, Rclone, and Thanos.
```

Comment:
I do not know if not renders correctly here :thinking: 
![image](https://github.com/elastisys/compliantkubernetes-apps/assets/12862587/c8736b13-3aca-459e-8cbc-8ad31d6114a8)

Reply:
Do you have the latest changes in public docs?
I added a postfix to unescape those.

Reply:
That was why :+1: